### ISSUE TITLE
please consider adding notifications event

### ISSUE THREAD

**Issue by ccoVeille** [maaslalani/pom#2]
**Describe the solution you'd like**

Your tool is simple, and it's great. But it requires user to look at the terminal where `pom` is launched to know if the period is over.

By notifications, I mean desktop notifications (like `notify-send` would emit).

**Additional context**

A project like this could be used to send the notifications
https://github.com/gen2brain/beeep

**Comment by maaslalani** [comment]
Yes! This is a great idea, I was also thinking about breaking up the focus and break times so you could use whichever notification daemon you want instead:

```bash
pom 25m && notify-send "Work done" \
  && pom --break 5m && notify-send "Break done"
```

**Comment by ccoVeille** [comment]
Look at beeep, it doesn't depend on a notifier, pure dbus, but yes unix command could be a simple solution

**Comment by danriedl** [comment]
Thank you, @maaslalani, for this fantastic tool! I've been using it for a few days now. However, I've also noticed that it lacks a feature for notifications. Currently, I keep pom open in a floating Zellij pane, but it loses focus when I switch to another pane.

**Comment by ccoVeille** [comment]
I agree, no matter the implementation, either in the tool via a library, or via documentation as in the example @maaslalani gave, there is a need
